### DISCUSSION TITLE
Number of regions option in Particle refinement block?

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [nextpyp/.github#56] [Q&A] [answered]
Hello, what does `Number of regions` option in Particle refinement block mean? For example the tomography tutorial step 10 (Constrained refinement) has `Set Number of regions to 8,8,2`. What does this do?

**Comment by abartesaghi** [comment, marked as answer]
This is described in the section **Region-based constrained projection matching** from the Nat. Meth. [paper](https://www.nature.com/articles/s41592-023-02045-0). `8,8,2` means that you will split the tomogram into that many subregions in x, y and z.
